### DISCUSSION TITLE
Some imported photo missing in the UI

### DISCUSSION THREAD

**Discussion by azukaar** [photoprism/photoprism#4598] [Q&A] [answered]
PLEASE PROCEED ONLY IF YOU ARE ABSOLUTELY SURE THAT THIS IS NOT A TECHNICAL SUPPORT INCIDENT AND/OR POSSIBLY A PROBLEM WITH SOME OTHER SOFTWARE YOU ARE USING. VISIT <https://www.photoprism.app/kb/getting-support> TO LEARN MORE ABOUT OUR SUPPORT OPTIONS. THANK YOU FOR YOUR CAREFUL CONSIDERATION!

---------------------------------------------------------------------------

#### 1. What is not working as documented?

Despite being properly imported, some photos do not show in the UI.
I can see these pictures in the Photoprism folder post-import (in the library/2024/05 folder)

![image](https://github.com/user-attachments/assets/e626d164-2918-41d8-b9df-418a40a52ac6)

But inside the UI, if i do calendar > April 2024 I only see:

![image](https://github.com/user-attachments/assets/cb6fed0e-ff53-4099-961d-cd2b0f1cc26a)

#### 2. How can we reproduce it?

no idea how to reproduce it... Here is one of the many missing pictures for reference, but I doubt it's because of the picture itself...

![20240410_214638_A1C8D007](https://github.com/user-attachments/assets/ddfb682a-330a-4df9-a0c9-d9b9791f2aa0)

#### 3. What behavior do you expect?

Simply for the imported pics to show

#### 4. What could be the cause of your problem?

I am assuming you only read the picture list from the DB and if it has a bug it coudl be missing pictures that have been imported 

#### 5. Can you provide us with example files for testing, error logs, or screenshots?

see before for picture. Log wise there's nothing relevant aside from "file is a duplicate" when trying to re-import it (which is correct)

```
 time="2024-10-18T13:39:02Z" level=info msg="media: 20240405_154808.jpg was taken at 2024-04-10 21:40:36 +0000 UTC (file mod time)"
2024-10-18 13:39:02
 time="2024-10-18T13:39:02Z" level=info msg="import: 20240405_154808.jpg is identical to 2024/04/20240410_214639_25A95F37.jpg (sha1 5d45a7dafe78f832242957d2f69c6989a571550d)"
```
#### 6. Which software versions do you use?

[PhotoPrism® CE](https://photos.yann-server.com/library/about)
Build [240915-e1280b2fb](https://docs.photoprism.app/release-notes/)

(b) mariadb 

(c) Debian

(d) Firefox

You can find the version/build number of the app in *Settings* by scrolling to the bottom. Note that MySQL 8 support has been discontinued, see system requirements at <https://docs.photoprism.app/getting-started/#system-requirements>.

*Always provide database and operating system details if it is a backend, import, or indexing issue. Should it be a frontend issue, at a minimum we require you to provide web browser and operating system details. When reporting a performance problem, we ask that you provide us with complete information about your environment, as there may be more than one cause.*

#### 7. On what kind of device is PhotoPrism installed?

This is especially important if you are reporting a performance, import, or indexing issue. You can skip this if you're reporting a problem you found in our public demo, or if it's a completely unrelated issue, such as incorrect page layout.

(a) AMD CPU

(b) 128gb

(c) HDD

(d) Docker

*Always provide device, memory, and storage details if you have a backend, performance, import, or indexing issue.*

**Comment by graciousgrey** [comment]
You can find our troubleshooting checklist for missing pictures here:

- https://docs.photoprism.app/getting-started/troubleshooting/#missing-pictures

Since the logs show that the file has no taken at date (only a file modification date), I assume that it is under review. Because of the missing date, it will not show up in the calendar until you approve it and add a date.

**Reply by azukaar** [reply]
Sorry what do you mean by "approve it", is there a place where I can see those and confirm the date? Or are you saying I have to go through all photos one by one to find ones with missing dates and add them manually?

**Reply by graciousgrey** [reply, marked as answer]
In case the photo is in review you would need to approve it or disable the quality filter so that it shows up in e.g. the search.

- https://docs.photoprism.app/user-guide/organize/review/

**Reply by azukaar** [reply]
![image](https://github.com/user-attachments/assets/35e9e751-3d2a-4f9c-b4e9-0feb412a2f5b)

oh Jesus I never knew that existed... Well, better late than never!
thanks

**Comment by lastzero** [comment]
We have added a note to our User Guide to better explain how this view works:
- https://docs.photoprism.app/user-guide/organize/calendar/

**Reply by lastzero** [reply]
In addition, our User Guide FAQs and the related Troubleshooting Checklist have been updated to include this information:
- https://docs.photoprism.app/getting-started/troubleshooting/#missing-pictures
- https://docs.photoprism.app/user-guide/faq/#media-library
